src/components/App.tsx

```diff
@@ -9,7 +9,7 @@ import {Navbar} from './nav/Navbar'
 import {GlobalStyle} from './globalStyle'
 import {MovieModalView} from './generic/movieModal/MovieModalView'
 
-export const App = () => {
+export const App = (): JSX.Element => {
```

Comment:
Let's pair on this. There is a rule to disable this, but I forgot which. It can be annoying to always specify the return type every time.